frontend/src/containers/LearnMore.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import LoginTopBar from '../containers/LoginTopBar';
+import PublicFooter from '../components/PublicFooter';
+import Grid, { Column } from '../components/Grid';
+
+export class LearnMore extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className='LearnMore'>
+          <LoginTopBar />
+          <div className='LearnMore-container'>
+            <section className='LearnMore-Section1'>
+              <div className='LearnMoreHeader'>Discover a new form of association</div>
+              <div className='LearnMoreHeader -faded'>Transparent by design.</div>
+              <div className='-img'></div>
+            </section>
+            <section className='LearnMore-Section2'>
+              <div className='LearnMoreHeader'>Open Your Finances To Your Community</div>
+              <Grid className='Wrap900'>
+                <Column span='6'>
+                  <div className='LearnMoreLead'>Unpaid Expenses</div>
+                  <div className='LearnMoreSubLead'>All submited expenses can bee seen by everyone and be marked as “pending” until you approve or reject them.</div>
```

Comment:
All submitted expenses can be seen by everyone. Core members of a collective can approve or reject them.